assets/js/lib/test-utils/factories/clusters.js

```diff
@@ -67,6 +67,7 @@ export const hanaClusterDetailsFactory = Factory.define(() => {
     stopped_resources: clusterResourceFactory.buildList(2),
     system_replication_mode: 'sync',
     system_replication_operation_mode: 'logreplay',
+    maintenance_mode: false,
```

Comment:
By the way, we should include this `maintenance_mode` entry in the `ascsErsClusterDetailsFactory` as well

Reply:
done